nibabel/streamlines/array_sequence.py

```diff
@@ -287,7 +277,11 @@ def _resize_data_to(self, n_rows, build_cache):
         if self._data.size == 0:
             self._data = np.empty(new_shape, dtype=build_cache.dtype)
         else:
-            self._data = _safe_resize(self._data, new_shape)
+            try:
+                self._data.resize(new_shape)
+            except ValueError:
+                self._data = self._data.copy()
+                self._data.resize(new_shape)
```

Comment:
```suggestion
                self._data.resize(new_shape, refcheck=False)
```

Reply:
Actually, do we need `refcheck=False` ? In theory, we just did a copy on the line before. Maybe we can replace those two lines with `self._data = np.resize(self._data, new_shape)`?.

Reply:
Oh. Just saw the errors on Travis (but some of them passed?!).

Reply:
I believe the reason we were doing this is because of the zero-fill behavior of `ndarray.resize`:

```
numpy.resize = resize(a, new_shape)
    Return a new array with the specified shape.
    
    If the new array is larger than the original array, then the new
    array is filled with repeated copies of `a`.  Note that this behavior
    is different from a.resize(new_shape) which fills with zeros instead
    of repeated copies of `a`.
```

Reply:
Ah right. Ok. I'll add the refcheck=False, seems for some systems it is needed. I believe it is safe to do so, since we are doing a copy just before (so in theory, there should be no references).